src/app/settings.py

```diff
@@ -143,3 +143,10 @@
 
 AUTH_USER_MODEL = "users.User"
 
+# Sentry
+
+if not DEBUG:
```

Comment:
Something like this?
```suggestion
if env.bool("ENABLE_SENTRY", False):
```

Reply:
yeah